Detectors/TOF/base/include/TOFBase/Geo.h

```diff
@@ -184,7 +196,35 @@ class Geo
   static constexpr Float_t BARS1[3] = { BARS[0], BAR1[1], BAR1[2] };                               // (cm)
   static constexpr Float_t BARS2[3] = { BARS[0], BAR2[1], BAR2[2] };                               // (cm)
 
+  static constexpr Float_t HHONY = 1.0;           // height of HONY Layer
+  static constexpr Float_t HPCBY = 0.08;          // height of PCB Layer
+  static constexpr Float_t HRGLY = 0.055;         // height of RED GLASS Layer
+  static constexpr Float_t HFILIY = 0.125;        // height of FISHLINE Layer
+  static constexpr Float_t HGLASSY = 0.160 * 0.5; // semi-height of GLASS Layer
+  static constexpr Float_t HCPCBY = 0.16;         // height of PCB  Central Layer
+  static constexpr Float_t WHONZ = 8.1;           // z dimension of HONEY Layer
+  static constexpr Float_t WPCBZ1 = 10.64;        // z dimension of PCB Lower Layer
+  static constexpr Float_t WPCBZ2 = 11.6;         // z dimension of PCB Upper Layer
+  static constexpr Float_t WCPCBZ = 12.4;         // z dimension of PCB Central Layer
+  static constexpr Float_t WRGLZ = 8.;            // z dimension of RED GLASS Layer
+  static constexpr Float_t WGLFZ = 7.;            // z dimension of GLASS Layer
+  static constexpr Float_t HSENSMY = 0.0105;      // height of Sensitive Layer
+
  private:
+  static void Init();
+
+  static Int_t GetSector(const Float_t* pos);
+  static Int_t GetPlate(const Float_t* pos);
+  static Int_t GetPadZ(const Float_t* pos);
+  static Int_t GetPadX(const Float_t* pos);
```

Comment:
in principle our coding guidelines ask for lower case letters for function names (which will be enforced at some moment); the start of the function should ideally be an action (verb)